### ISSUE TITLE
Fix Multiplane aspect bit Immutable Sampler test

### ISSUE THREAD

**Issue by spencer-lunarg** [KhronosGroup/Vulkan-ValidationLayers#5591]
https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/5583 broke up the test into just `VkLayerTest.MultiplaneAspectBits` but currently it is not testing anything

The issue seems to be with how `IsImmutableSampler()` is returning in the validation check that it doesn't hit `VUID-VkDescriptorImageInfo-sampler-01564`

Need to investigate more if this is a test bug or layer bug

**Comment by ziga-lunarg** [comment]
I think this is a VUID that is impossible to hit without also hitting another VUID.

**Comment by spencer-lunarg** [comment]
agree, I created https://gitlab.khronos.org/vulkan/vulkan/-/issues/3663 to see if this is a VU by accident then
